### ISSUE TITLE
Release version 0.0.42. 

### ISSUE THREAD

**Issue by bigeasy** [bigeasy/timezone#235]
Once again; here is the release checklist mentioned in #140.

- [ ] Ensure that all the features, including updated TZ databases have already been committed to master before performing a release. Do not add new code in a release commit.
- [ ] Ensure that your change log is up to date with all the changes prior to `0.0.42`. Do not edit the change log a bunch in a release commit.
- [ ] Create a `git checkout -b release-0.0.42` branch.
- [ ] Checkout your `release-0.0.42` branch.
- [ ] Update the version number in `src/timezone.js` to `0.0.42`.
- [ ] Update the version number in `t/version/get.t.js`.
- [ ] Update the version number in `package.json`.
- [ ] Ensure that the `CHANGELOG` looks reasonable.
- [ ] Run `make clean && make zic && make && npm test`.
- [ ] Run `git commit -m 'Release 0.0.42.' -a`.
- [ ] Get the date of the commit and add it to change log.
- [ ] Run `git commit --amend -a`.
- [ ] Close this issue with that commit message.
- [ ] Run `git tag v0.0.42 && git push origin v0.0.42`.
- [ ] Merge `release-0.0.42` into `master` with `git checkout master && git splice release-0.0.42`.
- [ ] Push `master`.
- [ ] Wait on [Travis CI build](https://travis-ci.org/bigeasy/timezone) to pass.
- [ ] Run `make publish` which will clear out `.AppleDouble` and `.DS_Store` before publishing and publish the *build* directory.

Publish data files to `gh-pages`.

- [ ] Go to your checkout of the `gh-pages`.
- [ ] In the `code` submodule run `git fetch && git checkout v0.0.42 && git submodule update`.
- [ ] In the `code` submodule `make clean && make zic && make && npm test`.

**Comment by bigeasy** [comment]
Duplicate of #249.
